src/scripts/training/lightgbm_python/train.py

```diff
@@ -211,7 +213,7 @@ def run(self, args, logger, metrics_logger, unknown_args):
         # make sure the output argument exists
         if args.export_model and multinode_config.main_node:
             os.makedirs(args.export_model, exist_ok=True)
-            args.export_model = os.path.join(args.export_model, "model.txt")
+            args.export_model = os.path.join(args.export_model, args.model_filename )
```

Comment:
currently, this would fail if value not provided, set a default in the add_argument() call